tests/test_tryte_byte_conv.c

```diff
@@ -6,11 +6,16 @@
  * "LICENSE" at the root of this distribution.
  */
 
-#include "stdio.h"
-#include "string.h"
+#include <stdio.h>
+#include <string.h>
 #include "tryte_byte_conv.h"
+#include "unity.h"
 
-int main() {
+void setUp(void) {}
+
+void tearDown(void) {}
+
+void test_bytes_trytes_bytes_conv(void) {
   const char test_str[1024] = {48, 48, 48, 48, 48, 0,  48, 48, 48, 48, 48, 48, 48, 48,
```

Comment:
Should we use a more gneral test string for testing？

Reply:
@splasky what do you think?

Reply:
How about a long text message or paragraph?

Reply:
I think it should be int array but not a string, since char doesn't a lot of numbers that int covers.

Reply:
However, the function of **bytes_to_trytes**, which uses `test_str` as input in the testing, requires the data type `unsigned char *`.

Reply:
I think it would better to generate a bigger random uint8_t array for testing.